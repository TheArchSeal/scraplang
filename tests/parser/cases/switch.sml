switch (x) {
    case 0:
        x;
    default:
    case 0:
        x;
}
