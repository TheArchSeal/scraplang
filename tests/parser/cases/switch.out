stmt (7):1:1 switch
    expr (3):1:9 x
    expr (3):2:10 0
    stmt (2):3:9 {}
        stmt (3):3:9 ;
            expr (3):3:9 x
    default
    stmt (2):5:5 {}
    expr (3):5:10 0
    stmt (2):6:9 {}
        stmt (3):6:9 ;
            expr (3):6:9 x
